### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by kripken**
More docs on quoting.

Mention SharedArrayBuffer is disabled in browsers for now due to Spectre.

### CODE REVIEW

File: site/source/docs/porting/pthreads.rst [hunk 1]
```diff
@@ -4,9 +4,11 @@
...
+Emscripten has support for multithreading using the new SharedArrayBuffer capability in browsers. That API allows sharing memory between the main thread and web workers as well as atomic operations for synchronization, which enables Emscripten to implement support for the Pthreads (POSIX threads) API.
 
-The proposed specification allows Emscripten applications to share the main memory heap between web workers. This along with primitives for low level atomics and futex support enables Emscripten to implement support for the Pthreads (POSIX threads) API.
+Support in asm.js is fairly stable in Emscripten. Support in WebAssembly is arriving in browsers, and still experimental.
+
+.. note:: Due to the Spectre set of vulnerabilities, browsers have disabled SharedArrayBuffer support for now. You can still experiment with it if you flip a pref in your browser.
```
Comment (saschanaz) [inline site/source/docs/porting/pthreads.rst]: Maybe worth adding *as of January 2018*.

Comment (kripken) [reply site/source/docs/porting/pthreads.rst]: Very good idea, done.

**Comment by kripken** [comment]
Also updated the faq entry on when it is safe to call compiled code.

**Comment by kripken** [comment]
Also noticed we don't mention wasm on the main page, fixed.

**Pull request merged**